src/sentry/static/sentry/app/views/projectFilters.jsx

```diff
@@ -227,16 +232,23 @@ const ProjectFiltersSettingsForm = React.createClass({
   mixins: [ApiMixin, ProjectState],
 
   getInitialState() {
+    let features = this.getProjectFeatures();
     let formData = {};
     for (let key of Object.keys(this.props.initialData)) {
```

Comment:
imo, this would be a bit clearer if we used a functional style i.e.

```
Object.keys(this.props.initialData).filter(...).forEach((key) => {
  formData[key] = this.props.initialData[key];
});
```

Reply:
Cool, I can change that